rllib/algorithms/cql/torch/cql_torch_learner.py

```diff
@@ -144,15 +150,24 @@ def compute_loss_for_module(
         # Calculate the TD error.
         td_error = torch.abs(q_selected - q_selected_target)
         # TODO (simon): Add the Twin TD error
+        if config.twin_q:
+            td_error += torch.abs(q_twin_selected, q_selected_target)
```

Comment:
Should this be `torch.minimum`?

Reply:
Sorry, meant: Should this be `torch.abs(q_twin_selected - q_selected_target)`

Reply:
No, this is fine like this. We only add the TD-error. The Q-values are computed by using the minimum of Q- and twin-Q-value. In case of the TD error we would actually cancel out large errors, if we would do so. But I am open to discussion.